lib/inputstreamhelper.py

```diff
@@ -801,9 +801,13 @@ def _check_drm(self):
 
     def _install_inputstream(self):
         """Install inputstream addon."""
```

Comment:
Thanks! This must have been a merge issue, as this was properly tested.

Travis would have caught this too 😉

Reply:
No, it was not a merge issue. I didn't see the error in the sanity tests, I did a manual test with a LibreELEC virtual machine. This problem occurs **after** `xbmc.exeutebuiltin` fails to install the inputstream add-on. Later on in the script inputstreamhelper is doing a version check on an unexisting add-on and that causes a crash.

Reply:
On my cellphone it appeared as if the try-statement was missing in my code.
That's why I thought this was basically a syntax error in the first place.